File: WebApp/Models/MI/MI_UNITEXCH.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class MI_UNITEXCH : JCLib.Mvc.BaseModel
    {
        public string MMCODE { get; set; }
        public string UNIT_CODE { get; set; }
        public string AGEN_NO { get; set; }
        public string EXCH_RATIO { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }

        // =================== 湘倫 ======================
        public string MMNAME_C { get; set; }
        public string UI_CHANAME { get; set; }
        public string AGEN_NAMEC { get; set; }
        // ===============================================
    }
}

File: WebAppVen/Scripts/app/model/MI_UNITEXCH.js
Ext.define('WEBAPP.model.MI_UNITEXCH', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MMCODE', type: 'string' },
        { name: 'UNIT_CODE', type: 'string' },
        { name: 'AGEN_NO', type: 'string' },
        { name: 'EXCH_RATIO', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        { name: 'MMNAME_C', type: 'string' },
        { name: 'UI_CHANAME', type: 'string' },
        { name: 'AGEN_NAMEC', type: 'string' }
    ]
});